.github/workflows/execution-performance.yaml

```diff
@@ -21,4 +23,5 @@ jobs:
     with:
       GIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
       RUNNER_NAME: executor-benchmark-runner
-      RUN_ONLY_SINGLE_NODE_PERF: ${{ github.event_name != 'schedule' }} # Run all tests on the scheduled cadence
+       # Run all tests only on the scheduled cadence, or explicitly requested
+      IS_FULL_RUN: ${{ github.event_name != 'schedule' || contains(github.event.pull_request.labels.*.name, 'CICD:run-execution-performance-full-test') }}
```

Comment:
Silly question: should this be inverted? i.e., previously we ran the single node tests iff it wasn't a scheduled run (e.g., a PR run). But, now, we're saying, run all the tests iff it's a scheduled run, or it's been requested, right? 😄

Reply:
argh, you are right :)